launcher/spec/launch_policy.go

```diff
@@ -35,7 +58,15 @@ func GetLaunchPolicy(imageLabels map[string]string) (LaunchPolicy, error) {
 
 	if v, ok := imageLabels[cmdOverride]; ok {
 		if launchPolicy.AllowedCmdOverride, err = strconv.ParseBool(v); err != nil {
-			return LaunchPolicy{}, fmt.Errorf("value of LABEL %s of the image is not a boolean %s", cmdOverride, v)
+			return LaunchPolicy{}, fmt.Errorf("value of LABEL %s of the image is not a boolean: %s, contact the image author", cmdOverride, v)
+		}
+	}
+
+	// default is debug only
+	if v, ok := imageLabels[logRedirect]; ok {
+		launchPolicy.AllowedLogRedirect, err = toLogRedirectPolicy(v)
+		if err != nil {
+			return LaunchPolicy{}, fmt.Errorf("value of LABEL %s of the image not valid: %v, contact the image author", logRedirect, err)
```

Comment:
let's simplify and add quotes around the label: "invalid image LABEL '%s'; contact the image author"

Reply:
done